cmd/tailscale/cli/configure_apple.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build darwin
+
+package cli
+
+import (
+	"context"
+	"errors"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+func init() {
+	maybeSysExtCmd = sysExtCmd()
+	maybeVPNConfigCmd = vpnConfigCmd()
+}
```

Comment:
these could be lazier. No need to do this work at init time (which can't be deadcode'd) 

if the global variables were instead `maybeSysExtCmd func() *ffcli.Command`

you could make a wrapper like https://go.dev/play/p/l-1vrzJgRVK and then do `ccall(maybeSysExtCmd)` instead of `maybeSysExtCmd`